docs/src/plots/activity_based_deposition.jl

```diff
@@ -0,0 +1,87 @@
+import Plots as PL
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CloudMicrophysics.Common as CO
+import CloudMicrophysics.HetIceNucleation as IN
+import CloudMicrophysics.Parameters as CMP
+
+FT = Float64
+const tps = TD.Parameters.ThermodynamicsParameters(FT)
+const feldspar = CMP.Feldspar(FT)
+const ferrihydrite = CMP.Ferrihydrite(FT)
+const kaolinite = CMP.Kaolinite(FT)
+
+# Initializing
+Δa_w = range(0, stop = 0.32, length = 50)    # difference in solution and ice water activity
```

Comment:
```suggestion
Δa_w = range(FT(0), stop = 0.32, length = 50)    # difference in solution and ice water activity
```
You might need something like this to make `Δa_w` `Float32`